tests/system/small/operations/test_datetimes.py

```diff
@@ -266,3 +266,20 @@ def test_dt_strftime_time():
         bf_result, expected_result, check_index_type=False, check_dtype=False
     )
     assert bf_result.dtype == "string[pyarrow]"
+
+
+@pytest.mark.parametrize(
+    ("col_name",),
+    DATETIME_COL_NAMES,
+)
+@skip_legacy_pandas
+def test_dt_normalize(scalars_dfs, col_name):
+    scalars_df, scalars_pandas_df = scalars_dfs
+    bf_series: bigframes.series.Series = scalars_df[col_name]
+    bf_result = bf_series.dt.normalize().to_pandas()
```

Comment:
we don't define `bf_series`, but just call `scalars_df[col_name].dt.normalize().to_pandas()`. Would that work also?

Reply:
True, bf_series is unnecessary. Fixed, thanks!